hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/scm/ReconPipelineFactory.java

```diff
@@ -48,8 +48,8 @@
 
     @Override
     public Pipeline create(HddsProtos.ReplicationFactor factor){
-      LOG.warn("Trying to create pipeline in Recon, which is prohibited!");
-      return null;
+      throw new RuntimeException(
```

Comment:
I think `UnsupportedOperationException` would be more appropriate.

Also, what about the other `create` method below?

Reply:
Good catch, will add it.